asab/api/doc.py

```diff
@@ -65,7 +65,9 @@ def build_swagger_specs(self):
 				},
 				"version": "1.0.0"
 			},
-			"servers": [{"url": "../../"}],  # Base path relative to openapi endpoint
+			"servers": [
+				{"url": "/", "description": "Here"}
```

Comment:
If there is this: https://github.com/TeskaLabs/asab/blob/a17f0d6e69d1cffe39069c4a0082db5543a91038/asab/api/doc.py#L108
what is this "servers" section good for?